src/airflow_docker/utils.py

```diff
@@ -1,30 +1,23 @@
+import functools
 import json
 import os
 
 from airflow.models import Variable
 
 
-def get_config(env=None, config_path="/airflow/dags", config_file="config.json"):
+@functools.lru_cache()
+def get_config(path="/airflow/dags", file="config.json"):
```

Comment:
Just realized, the path here should default to using the `[core]` `dags_folder` variable (via airflow configuration).